velox/docs/develop/aggregate-functions.rst

```diff
@@ -92,12 +92,18 @@ Fixed-width part of the accumulator is stored in the row. Variable-width
 part (if exists) is allocated using :doc:`HashStringAllocator <arena>` and a pointer is
 stored in the fixed-width part.
 
-A row is a contiguous byte buffer. Given N aggregates, first N / 8 bytes
-store the null flags, one bit per aggregate, followed by fixed-width
-accumulators.
-
-.. image:: images/aggregation-layout.png
-  :width: 600
+A row is a contiguous byte buffer.  Data is stored in the following order:
```

Comment:
It would be nice to clarify that sometimes there are gaps due to alignment. It would be helpful to explain where alignment boundaries are.